content/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -3,7 +3,11 @@
 Follow this guide to setup Google Drive as your <abbr>DVC project</abbr>'s
 [remote storage](/doc/command-reference/remote).
 
-## Configure a Google Cloud project
+## Configure a Google Cloud project (Highly recommended optional step)
```

Comment:
```suggestion
## Configure a Google Cloud project (highly recommended)
```

Reply:
I'm not sure what phrase fits the desired balance here - it seems we highly recommend to configure their own GCP project, but we should also highlight that the step is optional and user can save a lot time by skipping it.

Reply:
Recommended implies optional 🙂

Reply:
Also I added the term "optionally" in my next suggestion. Thanks